components/providers/AccordionLayout.vue

```diff
@@ -1,72 +1,113 @@
 <template>
   <article class="accordion-layout">
-    <cv-code-snippet
-      kind="oneline"
-      light
-      feedback-aria-label="Copy code snippet"
-    >
-      <!-- TODO: replace w/ dynamic code snippet -->
-      <span>pip install something</span>
-    </cv-code-snippet>
-    <img :src="image" class="accordion-layout__image">
     <p class="accordion-layout__description">
       {{ description }}
     </p>
-    <AppCta
-      kind="ghost"
-      v-bind="cta"
-    />
-    <!-- TODO: replace w/ dynamic code sample -->
-    <cv-code-snippet
-      kind="multiline"
-      light
-      feedback-aria-label="Copy code snippet"
-    >
-      import qiskit
-      from qiskit_runtime import ...
-
-      (very) short code example
-      Lorem ipsum dolor sit amet
-      consectetur adipiscing elit
-      Suspendisse id laoreet urna
-      nec egestas lorem
-      Donec porta
-      mauris sed facilisis pulvinar
-    </cv-code-snippet>
+    <img v-if="image" :src="image" class="accordion-layout__image">
+    <div class="accordion-layout__code-snippet">
+      <code class="accordion-layout__code-cell">
+        <pre class="accordion-layout__code-cell__line">{{ installation }}</pre>
+      </code>
+      <cv-button
+        title="Copy"
+        class="accordion-layout__copy-button"
+        @click="[copyToClipboard($event), $trackClickEvent(`Copy ${title} install code`, 'providers')]"
+      >
+        Copy
+      </cv-button>
+    </div>
+    <div class="accordion-layout__code-block">
+      <code class="accordion-layout__code-cell">
+        <template v-for="(line, index) in helloWorldExample">
+          <!-- eslint-disable vue/no-v-html -->
+          <pre v-if="line.startsWith('#')" :key="line" class="accordion-layout__code-cell__comment" v-html="line" />
+          <pre v-else :key="index" class="accordion-layout__code-cell__line" v-html="line" />
+          <!-- eslint-enable -->
+        </template>
+      </code>
+      <cv-button
+        title="Copy"
+        class="accordion-layout__copy-button"
+        @click="[copyToClipboard($event), $trackClickEvent(`Copy ${title} helloWorldExample`, 'providers')]"
+      >
+        {{ copyCodeCta }}
+      </cv-button>
+    </div>
```

Comment:
After creating the component I mentioned before, you can reuse it to replace this code too

Reply:
Updated in https://github.com/Qiskit/qiskit.org/pull/3000/commits/88ac621a65a1abc7d1e8637babbe6b846672e819